gui/jails/forms.py

```diff
@@ -230,13 +230,11 @@ def clean_jail_host(self):
 
     def clean_jail_mac(self):
         jail_mac = self.cleaned_data.get('jail_mac')
-        jobs = Jails.objects.all()
-        for jail in jobs:
-            if jail.jail_mac == jail_mac:
-                raise forms.ValidationError(_(
-                    "You have entered an existing MAC Address."
-                    "Please enter a new one."
-                ))
+        if is_jail_mac_duplicate(jail_mac):
+            raise forms.ValidationError(_(
+                "You have entered an existing MAC Address."
+                "Please enter a new one."
```

Comment:
It would be helpful to the user to tell them which other jail is already using that MAC address, something like:
"That MAC address is already in use in the _jailname_ jail.  Please enter a new, unique MAC address for this jail."